Cubical/Categories/CBPV/Base.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --lossy-unification #-}
+module Cubical.Categories.CBPV.Base where
+    open import Cubical.Categories.Category
+    open import Cubical.Foundations.Prelude
+    open import Cubical.Foundations.HLevels
+    open import Cubical.Categories.Monoidal.Base
+    open import Cubical.Categories.Presheaf
+    open import Cubical.Categories.Presheaf.Constructions
+    open import Cubical.Categories.Presheaf.Morphism.Alt hiding (_∘ˡ_)
+    open import Cubical.Categories.Limits.Terminal
+    open import Cubical.Categories.NaturalTransformation
+    open import Cubical.Categories.Functor
+    open import Cubical.Foundations.Structure
+    open import Cubical.Categories.Instances.Sets
+    open import Cubical.Foundations.Isomorphism
+    open import Cubical.Data.Sigma
+    open import Cubical.Categories.Enriched.Presheaf
+    open import Cubical.Categories.Monoidal.Enriched
+    open import Cubical.Categories.Enriched.More
+
+    open Category
+    open Functor
+    open NatTrans
+    open MonoidalCategory
+    open StrictMonCategory
+    open EnrichedCategory
+
+    record CBPVModel {ℓ ℓ' ℓS ℓE : Level} :
+      Type (ℓ-suc (ℓ-suc (ℓ-max ℓ (ℓ-max ℓ' (ℓ-max ℓS ℓE))))) where
+        field
+            𝓒 : Category ℓ ℓ'
+        open model 𝓒 {ℓS} using (self ; 𝓟Mon)
```

Comment:
completely unclear what `model` means here, definitely want a better name for that module.